### ISSUE TITLE
Comments in dget38.f look wrong. (only cosmetic one)

### ISSUE THREAD

**Issue by nakatamaho** [Reference-LAPACK/lapack#587]
**Description**

* DGEHRD reduces a general matrix A to upper Hessenberg form
* DHSEQR computes eigenvalues of a matrix of Hessenberg form and the Schur form

but in dget38.f, comments are only "Compute Schur form" 
--- lapack-3.9.1/TESTING/EIG/dget38.    2021-03-26 03:25:15.000000000 +0900
+++ lapack-3.9.1/TESTING/EIG/dget38.f   2021-06-21 15:51:04.000000000 +0900
@@ -194,7 +194,7 @@
      $      VMUL = ONE
          CALL DLACPY( 'F', N, N, T, LDT, TSAV, LDT )
 *
-*        Compute Schur form
+*        Compute upper Hessenberg form of matrix
 *
          CALL DGEHRD( N, 1, N, T, LDT, WORK( 1 ), WORK( N+1 ), LWORK-N,
      $                INFO )
@@ -210,7 +210,7 @@
          CALL DORGHR( N, 1, N, Q, LDT, WORK( 1 ), WORK( N+1 ), LWORK-N,
      $                INFO )
 *
-*        Compute Schur form
+*        Compute eigenvalues and the Schur form T
 *
          CALL DHSEQR( 'S', 'V', N, 1, N, T, LDT, WR, WI, Q, LDT, WORK,
      $                LWORK, INFO )

**Checklist**

- [x] I've included a minimal example to reproduce the issue
- [x] I'd be willing to make a PR to solve this issue

**Comment by langou** [comment]
Yes. OK. (1) Please submit a PR. (2) Can you check as well for s, c, and z arithmetic then? (3) "Compute upper Hessenberg form of matrix" is fine. We could also write: "Reduce to upper Hessenberg form". Either way is fine. Cheers, Julien.
